integ/integ.sh

```diff
@@ -56,15 +67,8 @@ clean_s3() {
 }
 
 if [ "${1}" = "cloudwatch" ]; then
-	mkdir -p ./integ/out
-	touch ./integ/out/cloudwatch-test
 	test_cloudwatch
 	clean_cloudwatch
-	if [ -f ./integ/out/cloudwatch-test ]; then
```

Comment:
Can the check remain after the call to `clean_cloudwatch`?

Because this change means that now if the test fails then clean does not run